reviews/module.py

```diff
@@ -0,0 +1,314 @@
+import discord
+from discord.ext import commands
+
+from pie.utils.objects import ScrollableVotingEmbed, VotableEmbed
+from pie import check, i18n, logger, utils
+
+from .database import Review, Subject
+
+_ = i18n.Translator("modules/school").translate
+guild_log = logger.Guild.logger()
+
+
+def _split_review(review: str) -> list[str]:
```

Comment:
The syntax `list[...]` is not supported in Python version 3.8. It needs to be rewritten with the correct syntax and imports, as we discussed on Discord.